decidim-stats/app/jobs/decidim/stats/stats_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Stats
+    class StatsJob < ApplicationJob
+      queue_as :default
+
+      def perform(minimum_count, emails)
+        data = Decidim::Stats::Runner.new(minimum_count: minimum_count).run
```

Comment:
We're having lots of memory issues, with this kind of processes. Let's keep it this way for now, but for the next developments we will have to use a temporary file where to dump the data while it is processed.

Reply:
If this keeps getting a problem we can try to work on it to get a better performance, sure!